release/nightly_tests/dataset/autoscaling_cpu_compute.yaml

```diff
@@ -2,7 +2,7 @@
 cloud_id: {{env["ANYSCALE_CLOUD_ID"]}}
 region: us-west-2
 
-max_workers: 0
+max_workers: 10
```

Comment:
Not sure why this wasn't an issue before. Maybe because we override `max_workers` for the specific node type